.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,24 @@
+# Contributions
+
+We welcome contributions in the form of issues and pull requests.  We view the contributions and the process as the same for github and external contributors.
```

Comment:
Hard to see changes, look for it commit by commit: https://github.com/actions/toolkit/pull/383/commits/c962b3e65ab6f0b2090e16fac3de4cd382ce3060